commons/commons-test/src/main/kotlin/ebi/ac/uk/test/ExtSubmissionFactory.kt

```diff
@@ -0,0 +1,29 @@
+package ebi.ac.uk.test
+
+import ebi.ac.uk.extended.model.ExtProcessingStatus.PROCESSED
+import ebi.ac.uk.extended.model.ExtSection
+import ebi.ac.uk.extended.model.ExtSubmission
+import ebi.ac.uk.extended.model.ExtSubmissionMethod.PAGE_TAB
+import java.time.OffsetDateTime
+import java.time.ZoneOffset.UTC
+
+val basicExtSubmission = ExtSubmission(
```

Comment:
we have create many of this, eventually we will need to put in common and use the same one

Reply:
That's actually what I did with this change. I placed this one in commons-test and used it from everywhere else